Server/ConcordiaCurriculumManager/Services/CourseService.cs

```diff
@@ -386,4 +387,15 @@ public async Task<IEnumerable<Course>> GetCoursesBySubjectAsync(string subjectCo
     {
         return await _courseRepository.GetCoursesBySubjectAsync(subjectCode);
     }
+
+    public async Task<Course> PublishCourse(string subject, string catalog)
+    {

```

Comment:
Sorry, I forgot to explain a detail. First you need to load the currently published version of the course, unpublish it, and then do this part here. That way there will only ever be at most 1 published version of a course, which is what we want.

The way the code is now, the old version of the course is still marked as published in the database, which is incorrect.

Reply:
Sounds good, I will change it.

Reply:
This has been changed now.